packages/analytics-node/src/node-client.ts

```diff
@@ -0,0 +1,226 @@
+import { AmplitudeCore, Destination } from '@amplitude/analytics-core';
+import { NodeConfig, NodeOptions } from '@amplitude/analytics-types';
+import { Context } from './plugins/context';
+import { useNodeConfig } from './config';
+
+export class AmplitudeNode extends AmplitudeCore<NodeConfig> {
+  async init(apiKey: string, userId?: string, options?: NodeOptions) {
```

Comment:
`userId` may not be a valuable argument here in init since it may be different for each event

Reply:
Sure, will move it.